tests/test_scalyr.py

```diff
@@ -12,26 +12,32 @@
     import ScalyrAgent, SCALYR_CONFIG_PATH, TPL_NAME, JWT_REDACTION_RULE,\
     get_parser, get_sampling_rules, get_redaction_rules, container_annotation
 
-from .conftest import CLUSTER_ID, NODE, APPLICATION_ID, APPLICATION_VERSION, COMPONENT
-from .conftest import SCALYR_KEY, SCALYR_DEST_PATH, SCALYR_JOURNALD_DEFAULTS
+from .conftest \
+    import CLUSTER_ID, CLUSTER_ENVIRONMENT, CLUSTER_ALIAS, NODE, APPLICATION_ID, APPLICATION_VERSION, COMPONENT
+from .conftest import SCALYR_KEY, SCALYR_DEST_PATH, SCALYR_JOURNALD_DEFAULTS, SCALYR_DEFAULT_PARSER
+
+DEFAULT_ENV = {
+    'WATCHER_SCALYR_API_KEY': SCALYR_KEY,
+    'CLUSTER_ENVIRONMENT': CLUSTER_ENVIRONMENT,
+    'CLUSTER_ALIAS': CLUSTER_ALIAS,
+    'CLUSTER_NODE_NAME': NODE
+}
 
 ENVS = (
-    {
-        'WATCHER_SCALYR_API_KEY': SCALYR_KEY, 'WATCHER_SCALYR_DEST_PATH': SCALYR_DEST_PATH,
-        'WATCHER_SCALYR_CONFIG_PATH': '/etc/config'
-    },
-    {
-        'WATCHER_SCALYR_API_KEY': SCALYR_KEY, 'WATCHER_SCALYR_DEST_PATH': SCALYR_DEST_PATH,
-    },
-    {
-        'WATCHER_SCALYR_API_KEY': SCALYR_KEY, 'WATCHER_SCALYR_DEST_PATH': SCALYR_DEST_PATH,
-        'WATCHER_SCALYR_JOURNALD': 'true',
-    },
-    {
-        'WATCHER_SCALYR_API_KEY': SCALYR_KEY, 'WATCHER_SCALYR_DEST_PATH': SCALYR_DEST_PATH,
-        'WATCHER_SCALYR_JOURNALD': 'true', 'WATCHER_SCALYR_JOURNALD_WRITE_RATE': 1,
-        'WATCHER_SCALYR_JOURNALD_WRITE_BURST': 2,
-    },
+    {**DEFAULT_ENV, **{
+        'WATCHER_SCALYR_DEST_PATH': SCALYR_DEST_PATH, 'WATCHER_SCALYR_CONFIG_PATH': '/etc/config'}},
```

Comment:
Why do we need the second double star?
Wouldn't be the following enough?
```python
{**DEFAULT_ENV, 'WATCHER_SCALYR_DEST_PATH': SCALYR_DEST_PATH, 'WATCHER_SCALYR_CONFIG_PATH': '/etc/config'}}
```

Reply:
+ Isn't `WATCHER_SCALYR_DEST_PATH` also in every env so it could also go into `DEFAULT_ENV`?

Reply:
I missed WATCHER_SCALYR_DEST_PATH. Is now in DEFAULT_ENV.
And the second dict with ** is of course unnecessary.